nixos/modules/programs/mbrola.nix

```diff
@@ -0,0 +1,30 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.programs.mbrola;
+in
+{
+  options.programs.mbrola = {
+    enable = lib.mkEnableOption "linking mbrola speech synthesizer voices installed through [](#opt-environment.systemPackages)";
+
+    installDefaultVoices = lib.mkOption {
```

Comment:
I would have expected a package option instead of this one.

Reply:
The plan is to eventually split the mbrola-voices package: https://github.com/NixOS/nixpkgs/issues/226869